Added Description and Resources for Relay Modern (#1000)
kamranahmedse/developer-roadmap: content/roadmaps/100-frontend/content/119-graphql/101-relay-modern.md

# Relay modern

> Relay is a JavaScript framework for building data-driven React applications powered by GraphQL, designed from the ground up to be easy to use, extensible and, most of all, performant. Relay accomplishes this with static queries and ahead-of-time code generation.

Reviewer: ```suggestion
Relay is a JavaScript client used in the browser to fetch GraphQL data. It's a JavaScript framework developed by Facebook for managing and fetching data in React applications. It is built with scalability in mind in order to power complex applications like Facebook. The ultimate goal of GraphQL and Relay is to deliver instant UI-response interactions.
```

<ResourceGroupTitle>Free Content</ResourceGroupTitle>

> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://relay.dev/docs/'>Introduction to Relay modern</BadgeLink>

Reviewer: ```suggestion
<BadgeLink colorScheme='blue' badgeText='Official Website' href='https://relay.dev/'>Official Website</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://relay.dev/docs/'>Introduction to Relay modern</BadgeLink>
```

> # Relay modern

Reviewer: ```suggestion
# Relay Modern
```
